vip-api/src/main/java/fr/insalyon/creatis/vip/api/business/PipelineBusiness.java

```diff
@@ -72,17 +73,19 @@ public class PipelineBusiness {
     private Supplier<User> currentUserProvider;
     private final WorkflowBusiness workflowBusiness;
     private final ApplicationBusiness applicationBusiness;
+    private final PublicationBusiness publicationBusiness;
```

Comment:
To remove, it should not be used anymore

Reply:
done